share/composer/vhd.ks

```diff
@@ -43,10 +64,9 @@ kernel
 grub2
 
 chrony
-cloud-init
 
-# Uninstall NetworkManager, install WALinuxAgent
--NetworkManager
 WALinuxAgent
+python3
```

Comment:
Lets add a comment here above these two lines to say these are required by WALinuxAgent

Reply:
Fixed.

Reply:
If WALinuxAgent requires it, why isn't it actually requiring it?